e2e/test/scenarios/dashboard-filters/dashboard-filters-auto-apply.cy.spec.js

```diff
@@ -414,6 +416,98 @@ describe("scenarios > dashboards > filters > auto apply", () => {
       undoToast().should("not.exist");
     });
   });
+
+  describe("embeddings", () => {
+    beforeEach(() => {
+      cy.signInAsAdmin();
+    });
+
+    describe("public embeds", () => {
```

Comment:
I haven't added extensive embedding tests because most cases should be covered with other tests, but each embedding test tests that
1. We can still apply filters manually when `auto_apply_filters: false`
2. Doesn't show toast no matter what, because public and signed embed users won't have write access to dashboards